Payments getall settlementUtc and AccountType (#465)
MewsSystems/gitbook-connector-api: operations/payments.md

| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Time interval during which the [Payment](#payment) was updated. Required if no other filter is provided. |
| `ChargedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Time interval during which the [Payment](#payment) was charged. Required if no other filter is provided. |
| `ClosedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Time interval during which the [Payment](#payment) was closed. Required if no other filter is provided. |

> | `SettlementUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Availability blocks](#availability-block) was settled. |

Reviewer: ```suggestion
| `SettlementUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Payments](#payment) were settled. |
```
